### ISSUE TITLE
Deprecation of DEF in Cython

### ISSUE THREAD

**Issue by rmlmcfadden** [dfujim/mudpy#9]
See the guide for [Migrating from Cython 0.29 to 3.0](https://cython.readthedocs.io/en/stable/src/userguide/migrating_to_cy30.html#deprecation-of-def-if).

The only `DEF`s in `mudpy` were introduced as part of #4 to avoid using hard-coded values in the "friendly" C-function calls.

Eventually, we should replace the `DEF`s with one of the strategies recommended in the guide.

**Comment by dfujim** [comment]
I'm not quite sure what to do here other than hard code it. The suggestions in the link you provided don't work in this cae. For example, by replacing `DEF` with `cdef int` I get the following compile error at many points: 

```      cpdef get_title(int file_handle):
          """Returns python string. Max number of characters: TITLE_CHAR_SIZE."""
          cdef char title[TITLE_CHAR_SIZE]
                          ^
      ------------------------------------------------------------
      
      mudpy/mud_friendly_wrapper.pyx:399:20: Not allowed in a constant expression
```

Effectively, cython expected a constant in the array declaration, but by using `cdef` the variable is no longer constant. One possibility is that a different type of array is used. I hope not though, that would be quite a pain. For now I'm shelving this until it becomes a problem, hopefully some other solution is present by then.

**Comment by rmlmcfadden** [comment]
I agree - none of the suggestions (apart from reverting the hard-coded values) seem to work as drop-in replacements.

Waiting on a better solution sounds good.
